### ISSUE TITLE
Documentation or implementation error for `degreesLat` and `degreesLong`

### ISSUE THREAD

**Issue by iamthechad** [shashwatak/satellite-js#74]
In the README, there is a section that says:
```
//  Convert the RADIANS to DEGREES for pretty printing (appends "N", "S", "E", "W", etc).
var longitudeStr = satellite.degreesLong(longitude),
    latitudeStr  = satellite.degreesLat(latitude);
```

The code for `degreesLong` and `degreesLat`, however, only returns numerical values without any of "N", "S", "E", or "W" appended. I'm not sure if this is a documentation error or an implementation error.

If it's a documentation error, the TypeScript definitions should be changed so that `degreesLat` and `degreesLong` return `number` instead of `string`.

**Comment by ezze** [comment]
It's error in documentation. Probably, it's left after first serious refactoring in 2.0.0.

**Comment by ezze** [comment]
Fixed in 4.1.1.
